[CardMedia] Move object-fit to the core (#15166)
mui-org/material-ui: docs/src/pages/demos/cards/cards.md

{{"demo": "pages/demos/cards/ImgMediaCard.js"}}

> > ⚠️ The previous demo replies on `object-fit` for centering the image. It's not supported by IE 11.

Reviewer: ```suggestion
> ⚠️ The previous demo relies on `object-fit` for centering the image. It's not supported by IE 11.
```

Reviewer: Would it not be more correct saying CardMedia relies on `object-fit`?

Reviewer 2: 👍

{{"demo": "pages/demos/cards/ImgMediaCard.js"}}

> > ⚠️ In the previous demo, CardMedia relies on `object-fit` for centering the image. It's not supported by IE 11.

Reviewer: Does mentioning the previous demo add any information here?

Reviewer 2: The limitation only applies when `component="img"` is provided.

Reviewer: ```suggestion
> ⚠️ When `component="img"`, CardMedia relies on `object-fit` for centering the image. It's not supported by IE 11.
```
?

Reviewer 2: Better! @alltom Can you accept the change?

![Capture d’écran 2019-04-03 à 00 15 18](https://user-images.githubusercontent.com/3165635/55439657-95a39000-55a5-11e9-9f00-7d0923490da4.png)

Author: @oliviertassinari How do I accept the change?

Reviewer 2: With the commit suggestion button. But don't worry about it. It's taken into account now.
